.envs/test.env

```
@@ -95,4 +95,5 @@ EXPLORER_DEFAULT_CONNECTION=my_database
 
 DISABLE_CELERY_BEAT_SCHEDULE=True
 
-PGAUDIT_LOG_SCOPES='ALL'
+PGAUDIT_LOG_SCOPES='ALL, -MISC'
+PGAUDIT_LOG_TYPE = 'docker'
```

```suggestion
PGAUDIT_LOG_TYPE=docker
```

Should this be the same as the sample? I'm not sure if this needs the spaces/quotes.